src/patternfly/components/EmptyState/empty-state-title.hbs

```diff
@@ -0,0 +1,6 @@
+<h1 class="pf-c-empty-state__title{{#if empty-state-title--modifier}} {{empty-state-title--modifier}}{{/if}}"
```

Comment:
do we want to make this `<h1>`?  there could be many occassions where the empty state is small and there are many of them on a page

Reply:
Reverted this. I made it an `<h1>` tho since that's what all of our existing examples are, and added this to the docs:

```
| `.pf-c-empty-state__title` | `<div>`, `<h1>`, `<h2>`, `<h3>`, `<h4>`, `<h5>`, `<h6>` |  Creates the empty state title. **Required** |
```